lib/rabbit_feed/event_definitions.rb

```diff
@@ -44,6 +45,7 @@ def payload_contains &block
       end
 
       def field name, options
+        @sensitive_fields << name.to_s if options.delete(:sensitive)
         fields << (Field.new name, options[:type], options[:definition])
```

Comment:
I don't want to start a flame war about which is better, but mixing two idioms in the same method (`attr_reader` and `ivar` access) doesn't look quite right...